plugins/travis-ci/src/state/useBuildWithSteps.ts

```diff
@@ -16,14 +16,14 @@
 import { errorApiRef, useApi } from '@backstage/core';
 import { useCallback } from 'react';
 import { useAsyncRetry } from 'react-use';
-import { circleCIApiRef, GitType } from '../api/index';
+import { travisCIApiRef } from '../api/index';
 import { useAsyncPolling } from './useAsyncPolling';
 import { useSettings } from './useSettings';
 
 const INTERVAL_AMOUNT = 1500;
 export function useBuildWithSteps(buildId: number) {
   const [{ token, repo, owner }] = useSettings();
-  const api = useApi(circleCIApiRef);
+  const api = useApi(travisCIApiRef);
   const errorApi = useApi(errorApiRef);
 
   const getBuildWithSteps = useCallback(async () => {
```

Comment:
Does Travis CI have a concept of a build with steps or is that just a Circle CI feature? It would be good to support it if it exists.